examples/dry_run.sh

```diff
@@ -1,9 +1,12 @@
 #!/bin/sh
 set -e
 
-for dir in $(ls -d */); do
+for dir in */; do
+  echo "--- Executing example in \"${dir}\" -----------------"
   cd "${dir}";
-  npm install
-  npm run dryRun
-  cd ..
+  if [ -f "package.json" ]; then
```

Comment:
Let's keep this in sync with [dry_run.bat](https://github.com/CodeIntelligenceTesting/jazzer.js/blob/2a6643fbf73ac00c57a2b22efe989f0c2cba6924/examples/dry_run.bat) and add a similar check there.

Reply:
Good catch, done!